code/hamburger.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Menu</title>
+    <link rel="stylesheet" href="./style.css" />
+</head>
+<body>
+    <header class="hamburgermenu-header">
+        <a href="./index.html"><img class="logo-3" src="./photos/dentnews.png" alt="Dentnews Logo"></a>
+    </header>
+
+    <section class="nav-links">
```

Comment:
Since it's navigation part, we should use `nav` semantic tag to build this block of code. Let's fix this ;)